### DISCUSSION TITLE
Ability to edit schema in playgroup (for github issues minimal repo)

### DISCUSSION THREAD

**Discussion by Akxe** [prisma/prisma#21436] [Q&A] [answered]
### Question

If the page https://playground.prisma.io/examples/ would allow us users to edit the schema, it would allow us to submit minimal reproduction examples in a well-defined environment. It could then be required in Prisma bug reports to help you debug our issues.

**Comment by ludralph** [comment, marked as answer]
Hi @Akxe 👋 

Thank you for your suggestion. I've shared it with the team. 

Meanwhile, you can also use [Code Sandbox](https://codesandbox.io/) to reproduce your examples and send to us to debugging as well.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
